scripts/resolve_refs.py

```diff
@@ -99,9 +99,15 @@ def sync_reqs_pr(sources, repo_name, pull_number):
         raise ValueError("repo_name %s doesn't match any source spec" % repo_name)
 
     for word in pull.body.split():
-        if '#' not in word:
+        if '#' in word:
+            target_repo_name, target_pull_name = word.strip(',;').split('#', 1)
+        elif "github.com" in word and "pull/" in word:
+            owner, name, p, target_pull_name = word.strip(',;').split("/")[-4:]
```

Comment:
I would do some additional checking on here to make sure that malformed links can't crash the script

Reply:
Some regex or urllib.parse? I think the latter is always available and would be a lot cleaner to understand.

Reply:
just making sure there are actually 4 elements in that list before unpacking it

Reply:
Just make sure there are 4 elements in the list before unpacking it

Reply:
Done